packages/tanstackstart/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sentry/tanstackstart",
```

Comment:
should this rather be `@sentry-internal/tanstackstart` then? Or are we planning on instructing users to install both packages?

Reply:
I had thought the same, I opted to do it like this for now because I messed up because I already published the package, so it's out there and grabbing SEO. We definitely need to update the README for the package that's out there for now, so I think this is still fine - maybe I'll deprecate the package and go for the `internal` name in a future PR. Hope that makes sense. I wish I hadn't published the package already 😂 😭